.github/workflows/crowdin-action.yml

```diff
@@ -18,10 +18,10 @@ jobs:
     steps:
 
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v4
 
     - name: crowdin action
-      uses: crowdin/github-action@a3160b9e5a9e00739392c23da5e580c6cabe526d
+      uses: crowdin/github-action@v1
```

Comment:
You need to use a commit hash, and have it added to the whitelist.

Reply:
@Mrtenz thank you. Can you please link to the whitelist, and if there is any internal documentation as to how we do that?

Reply:
@i18nlaurel I will review with you separately.